src/explore-education-statistics-frontend/src/modules/data-catalogue/components/DataSetFileUsage.tsx

```diff
@@ -24,7 +24,7 @@ export default function DataSetFileUsage({
   tableToolLink,
   onDownload,
 }: Props) {
-  const downloadLink = `${process.env.CONTENT_API_BASE_URL}/data-set-files/${dataSetFileId}/download`;
+  const downloadLink = `${process.env.PUBLIC_URL}data-catalogue/data-set/${dataSetFileId}/csv`;
```

Comment:
Just checking that we can rely on `PUBLIC_URL` to always have the slash - I know there are cases with env variables where sometimes they do and sometimes they don't.

All that said, might be easiest to let it be tested as it's an easy fix if it isn't right. The only way we get stung by this is if prod is different to every other environment, which you'd hope isn't the case.

Reply:
Good point, I've cleaned that up to ensure we have correct slashes when creating the URL